ts/packages/anchor/src/nodewallet.ts

```diff
@@ -30,14 +30,18 @@ export default class NodeWallet implements Wallet {
     return new NodeWallet(payer);
   }
 
-  async signTransaction(tx: Transaction): Promise<Transaction> {
-    tx.partialSign(this.payer);
+  async signTransaction(
+    tx: VersionedTransaction
```

Comment:
I think this probably needs to be `Transaction | VersionedTransaction` everywhere so as not to break everyone?

Reply:
That does sound reasonable, I'll look into it.